vault/mount.go

```diff
@@ -1131,6 +1142,16 @@ func (c *Core) setupMounts(ctx context.Context) error {
 			return errLoadMountsFailed
 		}
 
+		// Initialize
+		if !nilMount {
+			c.postUnsealFuncs = append(c.postUnsealFuncs, func() {
+				err := backend.Initialize(ctx, &logical.InitializationRequest{Storage: view})
+				if err != nil {
+					c.logger.Error("failed to initialize mount entry", "path", entry.Path, "error", err)
+				}
+			})
+		}
```

Comment:
It feels better to me to log the fact that we got an error back from Initialize(), but I'd be glad to change it if you'd like

Reply:
We can still log, i was just meaning that we'd move this logic up to the previous postUnsealFunc that we specify

Reply:
I don't think so, because there are intervening errors that could cause the method to exit, like failing to mount at all.  In that case, I think we shouldn't call Initialize()

Reply:
Makes sense, however i think we should verify the postUnsealFuncs will be handled in the correct order. e.g. the func that sets storage to read-write needs to be called before the initialize call. I think since the other one is being deferred this will be appended to the list first. I assume we execute them in FIFO order?

Reply:
Scratch that- the above one isn't deferred.